en/docs/deploy-and-publish/publish-on-dev-portal/third-party-api-support.md

```diff
@@ -7,19 +7,23 @@ In previous versions of API Manager, third-party APIs were available when import
 
 ## Creating a Third-party API
 
+#### Method 1 - REST APIs
```

Comment:
Method 1 should be not REST APIs, rather the APIs already supported by APIM, and the method 2 should be async/streaming APIs not supported by APIM (such as AMQP, MQTT, etc)